dvc/api.py

```diff
@@ -43,13 +43,27 @@ class SummonError(DvcException):
     pass
 
 
+class UrlNotDvcRepoError(DvcException):
+    """Thrown if given url is not a DVC repository.
+
+    Args:
+        url (str): url to the repository.
```

Comment:
I don't see a `url` arg.
p.s. I'd capitalize "URL" in the explanation side.

Reply:
It's in `__init__(self, url)`, and is following Google-style docstrings. In Google-style, `__init__()` method can be both documented in class-level or inside `__init__` methods itself. I'll make a quick PR regarding the *URL* though.

Reply:
> It's in __init__(self, url)

🤦‍♂ sry didn't notice it's a class haha

> following Google-style docstrings

Great! Did you read my proposal to use that in https://github.com/iterative/dvc/issues/3176#issuecomment-576539623? Or is this a happy coincidence?

Don't worry about `url`, I'll just address all my own feedback here in a quick PR. Working on it now.